### ISSUE TITLE
Missing descriptive text in label elements for tooltip.

### ISSUE THREAD

**Issue by reeteshsingh93** [carbon-design-system/carbon-components-react#2317]
## Environment

> Operating system
MacOS Mojave

> Browser

Chrome

> Automated testing tool and ruleset

> Assistive technology used to verify

## Detailed description

> What version of the Carbon Design System are you using?

  "carbon-components": "9.57.0",
    "carbon-components-react": "6.54.1",
    "carbon-icons": "7.0.7",

> What did you expect to happen?

> What happened instead?

> What WCAG 2.1 checkpoint does the issue violate?

<AVT> Provide descriptive text in label elements

## Steps to reproduce the issue

1. Step one
2. Step two
3. Step three
4. etc.

> Please create a reduced test case in CodeSandbox
> https://codesandbox.io/s/github/IBM/carbon-components-react/tree/master/examples/codesandbox

## Additional information

- Screenshots or code
![image](https://user-images.githubusercontent.com/46310374/57198877-c7898700-6f95-11e9-843f-e18342d708dc.png)

- Notes
I tried to add some text in  label  here...., that pass the test(DAP violation)

**Comment by dakahn** [comment]
👋 Thanks for raising an issue. Please fill out the remainder of the issue template. Of special importance are A) specifying which DAP ruleset you're testing against and B) providing a reduced test case I can verify with.

**Comment by reeteshsingh93** [comment]
DAP ruleset: IBM Accessibility February 2019

**Comment by dakahn** [comment]
@reeteshsingh93 testing with that same ruleset against the Toggle component at v9.carboncomponents.com and I'm not getting that error.

If you could provide a test case where I can recreate what you're seeing that'd be cool 👍🏽
